sequencing/sequencer.go

```
@@ -281,10 +281,13 @@
 
 	db    *badger.DB // BadgerDB instance for persistence
 	dbMux sync.Mutex // Mutex for safe concurrent DB access
+
+	metrics *Metrics
```

golangci-lint

File is not `goimports`-ed with -local github.com/rollkit

(goimports)